build.json

```diff
@@ -1,18 +1,19 @@
 {
-  "image": "homeassistant/{arch}-builder",
-  "shadow_repository": "ghcr.io/home-assistant",
-  "build_from": {
-    "aarch64": "ghcr.io/home-assistant/aarch64-base:3.13",
-    "armv7": "ghcr.io/home-assistant/armv7-base:3.13",
-    "armhf": "ghcr.io/home-assistant/armhf-base:3.13",
-    "amd64": "ghcr.io/home-assistant/amd64-base:3.13",
-    "i386": "ghcr.io/home-assistant/i386-base:3.13"
-  },
-  "args": {
-    "VCN_VERSION": "0.9.6"
-  },
-  "labels": {
-    "io.hass.type": "builder",
-    "org.opencontainers.image.source": "https://github.com/home-assistant/builder"
-  }
-}
+    "image": "homeassistant/{arch}-builder",
```

Comment:
This file needs to be in here for the build to succeed since we need to use the :latest tag of an already build image.
Will do a follow-up when a new latest is pushed to remove this file.